frontend/src/components/pages/CampsList/CampsTable.tsx

```diff
@@ -195,10 +194,7 @@ const CampsTable = ({
                 }}
               >
                 {camp.campSessions.length > 0
-                  ? getFormattedCampDateRange(
-                      camp.campSessions[0].dates,
-                      camp.campSessions[camp.campSessions.length - 1].dates,
-                    )
+                  ? getFormattedCampDateRange(camp.campSessions)
```

Comment:
I edited the function to now take all sessions of a camp, and then sort them. I don't think we guarantee that the order of sessions as they appear in the object indicate order of what dates they are held on (especially considering overlapping sessions) so now I'm taking all sessions and sorting them all at once.